apps/angular/src/app/features/anime/anime-details-page/anime-details-page.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="anime">
+	<ng-container *ngIf="anime$ | async as anime">
+		<h2 class="anime__title">{{ anime.titleEnglish | empty }}</h2>
+		<h2 class="anime__title">{{ anime.titleJapanese | empty }}</h2>
+		<div class="anime__details">
+			<img
+				class="anime__details-image"
+				[alt]="anime.titleEnglish"
+				[src]="anime.imageUrl"
+				(click)="openDialog(anime.imageUrl, anime.titleEnglish)"
+			/>
+			<div>
+				<p><strong>Season:</strong> {{ anime.season | readable }}</p>
+				<p><strong>Synopsis:</strong> {{ anime.synopsis }}</p>
+				<p><strong>Airing:</strong> {{ anime.airing | yesOrNo | empty }}</p>
+				<p><strong>Date start:</strong> {{ anime.aired.start | date | empty }}</p>
+				<p><strong>Date end:</strong> {{ anime.aired.end | date | empty }}</p>
+				<p><strong>Genres:</strong> {{ anime.genresData | readable | empty }}</p>
+				<p><strong>Studios:</strong> {{ anime.studiosData | readable | empty }}</p>
+				<p><strong>Rating:</strong> {{ anime.rating | readable | empty }}</p>
+				<p><strong>Source:</strong> {{ anime.source | readable | empty }}</p>
+				<p><strong>Type:</strong> {{ anime.type | readable | empty }}</p>
+				<p><strong>Status:</strong> {{ anime.status | readable | empty }}</p>
+				<p><strong>Created:</strong> {{ anime.created | date | empty }}</p>
+				<p><strong>Modified:</strong> {{ anime.modified | date | empty }}</p>
```

Comment:
Revise formatting, please.
```suggestion
				<p><strong>Modified:</strong>{{ anime.modified | date | empty }}</p>
```

Reply:
I need this space.
With space: 
![image](https://github.com/saritasa-nest/camp-js-2023-egor-ryazanov/assets/89483642/24eddbce-8ea8-463d-bcd3-282a03c5f932)
Without space:
![image](https://github.com/saritasa-nest/camp-js-2023-egor-ryazanov/assets/89483642/60711771-ecf7-4e78-9f70-dfe091e05967)